.travis.yml

```diff
@@ -14,17 +14,15 @@ addons:
   apt:
     packages:
       - rpm
+  chrome: stable
 
 git:
   depth: 5
 
 language: node_js
 
 node_js:
-  - 10
-
-addons:
-  chrome: stable
```

Comment:
This should stay in - Chrome is needed to make the tests work.
(Travis is not used by the project at the moment due to the lack of free machine time for open source projects.)